drizzlepac/runastrodriz.py

```diff
@@ -463,6 +463,10 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
         if _calfiles_flc:
             updatewcs.updatewcs(_calfiles_flc, use_db=False, checkfiles=False)
 
+        # adds skycell keyword to science header of all flt(c) and drz(c,w) files. 
+        for filename in _calfiles+_calfiles_flc:
```

Comment:
Please add a note here that indicates the SKYCELL value for IPPPSSOOT and SVM products may be different as the current computation is based upon the WCSNAME of the input images.